docs/troubleshooting.md

```
@@ -71,6 +71,44 @@ SELECT * FROM pgedge_vectorizer.failed_items;
+A queue being throttled looks like this and needs no intervention:
+
+```sql
+SELECT status, count(*), max(attempts) AS attempts,
+       max(rate_limit_deferrals) AS deferrals
+  FROM pgedge_vectorizer.queue
+ GROUP BY status;
+```
+
+Each deferral is logged with the status code and the wait taken:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)